webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -141,7 +146,11 @@ class ManageManifestModal extends Component {
         url: 'http://redhat.com',
       },
     });
-
+    const buttonLoading = (
+      <span>
+        {__('Updating...')}
+        <span className="fa fa-spinner" />
```

Comment:
@waldenraines the `Spinner` component doesn't work inside buttons ( because it uses `div`  wrapper).

Reply:
Ah, okay.  Well now I'm seeing that the spinner isn't spinning at all.